### DISCUSSION TITLE
Pseudo element content with space not working

### DISCUSSION THREAD

**Discussion by IndranilBhattacharya** [tailwindlabs/tailwindcss#7957] [Help] [answered]
Suppose I have a pseudo element where I want to set its content attribute to "Log Out" using:

`after:content-['Log Out']`

But this throwing error. Instead same way this is working:

`after:content-['LogOut']`

How to solve this issue with content **text with space**?
Thanks in advance.

**Comment by adamwathan** [comment, marked as answer]
Here's the docs on this:

https://tailwindcss.com/docs/adding-custom-styles#handling-whitespace

**Reply by IndranilBhattacharya** [reply]
Thank you.
That solves my issue. 
But what if there is actually "_" required in content text?
